Assets/Project/Scripts/Modules/GamePlayScene/Bottle/DynamicBottleController.cs

```diff
@@ -77,64 +63,50 @@ protected override void Awake()
             _flickGesture = GetComponent<FlickGesture>();
             _flickGesture.MinDistance = 0.2f;
             _flickGesture.FlickTime = 0.2f;
+            Observable.FromEvent<EventHandler<EventArgs>, Tuple<object, EventArgs>>(h => (x, y) => h(Tuple.Create<object, EventArgs>(x, y)), x => _flickGesture.Flicked += x, x => _flickGesture.Flicked -= x)
```

Comment:
```suggestion
            Observable.FromEvent<EventHandler<EventArgs>, Tuple<object, EventArgs>>(h => (x, y) => h(Tuple.Create(x, y)), x => _flickGesture.Flicked += x, x => _flickGesture.Flicked -= x)
```
`Create`に型指定いらないだとRider様がおっしゃいました。

Reply:
6720515